cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryStreamAdmin.java

```diff
@@ -41,8 +41,8 @@ public InMemoryStreamAdmin(InMemoryQueueService queueService) {
   }
 
   @Override
-  public void dropAll() throws Exception {
```

Comment:
This is confirmed? Removing the ability to drop across namespaces? I think its fair. Should do it for queues as well. I'll take care of it.